.travis.yml

```diff
@@ -1,10 +1,18 @@
 sudo: false
 language: ruby
-cache: bundler
-rvm:
-    - 2.4.0
-before_install:
-  - gem update --system --no-doc
+dist: trusty
+rvm: 2.4.0
+cache:
+  bundler: true
+  directories: $TRAVIS_BUILD_DIR/tmp/.htmlproofer
+before_install: gem update --system --no-doc
+matrix:
+  include:
+    - os: osx
+      script: bundle exec rake proof_external
+      env: NOKOGIRI_USE_SYSTEM_LIBRARIES=true
+  allow_failures:
+    - script: bundle exec rake proof_external
```

Comment:
it means that only the separate build using the script `rake proof_external` is allowed to fail